static/src/stylesheets/print.scss

```diff
@@ -7,6 +7,7 @@
 .block-share__link,
 .l-header,
 .top-banner-ad-container,
+.adblock-sticky__message,
```

Comment:
Is this the best way to hide elements in print? I think it buries the behaviour somewhat. Could a print-hidden mixin allow us to group together the print behaviour with the rest of the component's behaviour?

I'd be thinking of a mixin like the following:

    @mixin print-hidden {
      @media print {
        & {
          display: none !important;
        }
      }
    }
    
    .hide-me-in-print {
      border: 1px solid blue;
      @include print-hidden
    }